CHANGES.txt

```diff
@@ -24,6 +24,11 @@ Features
 * Pass ``post`` object and ``lang`` to post compilers (Issue #2531)
 * Pass ``url_type`` into template's context.
 * Make thumbnail naming configurable with ``IMAGE_THUMBNAIL_FORMAT``.
+* There is a new plugin category ``Taxonomy`` which allows to easily
+  create new taxonomies. Some of the existing taxonomies (authors,
+  archives, indexes, page index, sections) have been converted to
+  the new system. Tags will be converted to the new system as soon
```

Comment:
Hmm, actually, that might not be too bad. I've tried implementing the special code (without testing it yet though), it looks acceptable so far. I'll handle the tags plugin next.